### DISCUSSION TITLE
Is the project dead ?

### DISCUSSION THREAD

**Discussion by Fred-Vatin** [PrismJS/prism#3834] [Q&A]
Hi,

There has not been new release since 2022 and there are 65 PR not merged, some of which add new languages support. Do you wait v2 ?

**Comment by mAAdhaTTah** [comment]
We have been discussing V2 on & off for the past few weeks so no, not dead. Just been tough to align on a direction with the maintainers being busy.

**Reply by james-gould** [reply]
Would you reconsider opening up PRs prior to that?

I know personally I have a few things I'd like to fix/extend for the `csharp` lexer, I'm sure others would as well. Not looking to pile onto your workload with that though! 

I'd rather contribute those changes to the repo instead of fixing it locally for my own purposes though.
